diesel/src/expression/mod.rs

```diff
@@ -110,20 +131,20 @@ impl<T: Expression> AsExpression<T::SqlType> for T {
 /// Columns will implement this for their table. Certain special types, like
 /// `CountStar` and `Bound` will implement this for all sources. All other
 /// expressions will inherit this from their children.
-pub trait SelectableExpression<QS>: Expression {
+pub trait SelectableExpression<QS>: AppearsOnTable<QS> {
```

Comment:
Nah, it's fine. Mentioning the relation to `AppearsOnTable` might've been nice, but it's a supertrait, so people should look at it anyway (and it has good docs).